manual/src/threelang/tfchain/tfchain.md

```diff
@@ -0,0 +1,237 @@
+# TF Chain Actions
+In a simple format you can do almost all the extrinsic and queries on the chian. List of the namespaces
+
+## Name spaces and their actions
+- [Account](#account)
+- [Balance](#balance)
+- [Twins](#twins)
+- [Nodes](#nodes)
+- [Farms](#farms)
+- [Contracts](#contracts)
+- [Service Contract](#service-contract)
+- [Metadata](#metadata)
+
+
+## Account
+- Create new account on the chain by Generating mnemonic
```

Comment:
Can you follow the same documentation formatting as done for the tfgrid 3lang documentation? It's better to have one uniform way of documenting things.